tests/contrib/openstack/test_policyd.py

```diff
@@ -236,11 +237,11 @@ def test_clean_policyd_dir_for(self,
                                    mock_shutil_rmtree,
                                    mock_os_remove,
                                    mock_log):
-        if six.PY3:
+        if hasattr(os, 'scandir'):
             mock_scan_dir_parts = (mock.patch, ["os.scandir"])
         else:
             mock_scan_dir_parts = (mock.patch.object,
-                                   [policyd, "_py2_scandir"])
+                                   [policyd, "_fallback_scandir"])
```

Comment:
This could get brittle if `_fallback_scandir` is ever changed in the future. Perhaps add a comment in that function that its relied on by the mocks?

Reply:
are you saying comment in `./contrib/openstack/policyd.py`  around `_fallback_scandir`?  To let a future dev know that this test is mocking it out?